### DISCUSSION TITLE
importing wild card to import secrets. Is it possible?

### DISCUSSION THREAD

**Discussion by igorpalant** [Azure/secrets-store-csi-driver-provider-azure#430] [Q&A]
Is it possible to inject secrets using wild characters. For example, I have service "myservice" running in "dev" and "test" k8s namespaces and I have secrets in Azure Key Vault stored in following manner:
dev.myservice.secret1=value123
dev.myservice.secret2=value345
test.myservice.secret1=value456
test.myservice.secret2=value678
When "myservice" starts in "dev" namespace, I want to inject all secrets into it that match following dev.myservice.*

We plan to have A LOT of secrets. If there are other ways to achieve this, please let me know. Any help is appreciated.

**Comment by aramase** [comment]
It's currently not possible to import secrets based on a wildcard. We have an issue in the backlog for this: https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/357. Please feel free to comment or +1 in the issue.
